### ISSUE TITLE
[Bug] [a11y] The logo cannot be selected and it is not read

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#6993]
## Steps to reproduce
1. Make sure you have a clean profile.
2. Turn talkback on.
3. Go to Firefox Preview and select the logo.

### Expected behavior
The logo is selected and presented.

### Actual behavior
The logo cannot be selected and it is not presented.

### Device information
* Android device: Samsung Galaxy Tab A T515 (Android 9), LG G7 Fit (Android 8.1)
* Fenix version: 3.0.0-beta.3 (#13392157), Nightly build from 12/05 (#13391806)

![20191206](https://user-images.githubusercontent.com/40694158/70310368-1e02c080-1818-11ea-8e53-fda103fdf27c.gif)

**Comment by ekager** [comment]
It looks like this was intentional because we had set `importantForAccessibility` to `no`. Is there true value for the user in reading this logo aloud? I think this might just clutter the function of the homescreen for these users?

**Comment by mcarare** [comment]
I thought it was set by mistake, because making it not focusable and not clickable already makes it not important for accessibility. Either way, the logo is readable in the About section. IMO, we should either make it in both or in none.

**Comment by sblatz** [comment]
ccing @MReschenberg to see her thoughts here.

**Comment by MReschenberg** [comment]
I’d vote to change the functionality to none in About; it doesn’t need to be focusable for SR users.

**Comment by sblatz** [comment]
Closing as expected behavior. Going to open a ticket to remove it from about :)
